icu4c/source/common/ucasemap.cpp

```diff
@@ -420,6 +426,96 @@ void toUpper(int32_t caseLocale, uint32_t options,
 
 #if !UCONFIG_NO_BREAK_ITERATION
 
+
+/**
+ * Input: c is a letter I with or without acute accent.
+ * start is the index in src after c, and is less than segmentLimit.
+ * If a plain i/I is followed by a plain j/J,
+ * or an i/I with acute (precomposed or decomposed) is followed by a j/J with acute,
+ * then we output accordingly.
+ *
+ * @return the src index after the titlecased sequence, or the start index if no Dutch IJ
+ * @throws IOException
+ */
+int32_t maybeTitleDutchIJ(const uint8_t *src, UChar32 c, int32_t start, int32_t segmentLimit,
```

Comment:
Please surround this function with the anonymous namespace.
Like the previous code, but it's probably easier to start a new anonymous-namespace block to avoid dealing with the partial overlap of that with the UCONFIG_NO_BREAK_ITERATION section.
Please end the anonymous namespace exactly like line 425, with that comment.

Reply:
= leftover from https://github.com/unicode-org/icu/pull/1869#discussion_r753461595

Reply:
Done.